components/hab/src/cli/hab/origin.rs

```diff
@@ -235,6 +238,60 @@ pub struct UploadGroup {
     public_file: Option<PathBuf>,
 }
 
+#[derive(ConfigOpt, StructOpt)]
+#[structopt(no_version)]
```

Comment:
You could make this `#[structopt(no_version, rename_all = "screamingsnake")]` to avoid the need to explicitly type `#[structopt(name = "<NAME">` on the fields below.